estela-api/api/views/auth.py

```diff
@@ -245,7 +252,9 @@ def get_parameters(self, request):
         return token, user_id
 
     @swagger_auto_schema(
-        methods=["POST"], responses={status.HTTP_200_OK: TokenSerializer()}
+        methods=["POST"],
+        responses={status.HTTP_200_OK: TokenSerializer()},
+        tags=["account"],
```

Comment:
I just used the paths defined in `urls.py` as tags https://github.com/bitmakerla/estela/blob/3cc024c57b2724e16aa9cdf9bf153076ca04bb02/estela-api/api/urls.py#L44-L58

for example `tags=["project", "spider", "job", "job-data"],` would have those tags because of the parts of the path     https://github.com/bitmakerla/estela/blob/3cc024c57b2724e16aa9cdf9bf153076ca04bb02/estela-api/api/urls.py#L35